1.React.createElement.md

```diff
@@ -0,0 +1,32 @@
+**Jam3 Lessons - React**
```

Comment:
I have no idea why this article is here. But let's leave it since I don't know why I lost it.